src/index.js

```diff
@@ -108,19 +108,23 @@ async function main() {
 
```

Comment:
@imstar15 
Please replace the time parameter with actual timestamps,
    `Fixed: { executionTimes: [0] }`

Reply:
Never mind, [0] is to execute immediately. No need to change.